desktop/core/src/desktop/js/apps/admin/Configuration/ConfigurationTab.test.tsx

```diff
@@ -0,0 +1,199 @@
+                value: 'Another value'
+              }
+            ]
+          }
+        ],
+        conf_dir: '/conf/directory'
+      })
+  })
+) as jest.Mock;
+
+describe('Configuration Component', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('Renders Configuration component with fetched data', async () => {
+    render(<Configuration />);
+
+    await waitFor(() => {
+      expect(screen.getByText(/Sections/i)).toBeInTheDocument();
+      expect(screen.getByText(/Desktop/i)).toBeInTheDocument();
+      expect(screen.getByText(/example\.config/i)).toBeInTheDocument();
+      expect(screen.getByText(/Example value/i)).toBeInTheDocument();
+    });
+  });
+
+  test('Filters configuration based on input text', async () => {
+    render(<Configuration />);
+
+    const filterInput = screen.getByPlaceholderText('Filter in desktop...');
+    fireEvent.change(filterInput, { target: { value: 'another' } });
+
+    await waitFor(() => {
+      expect(screen.queryByText('example.config')).not.toBeInTheDocument();
+      expect(screen.getByText('another.config')).toBeInTheDocument();
+    });
+  });
+
+  test('Displays message for empty configuration section', async () => {
+    global.fetch = jest.fn(() =>
+      Promise.resolve({
+        json: () =>
+          Promise.resolve({
+            apps: [{ name: 'desktop', has_ui: true, display_name: 'desktop' }],
+            config: [
+              { help: 'No help available.', key: 'desktop', is_anonymous: false, values: [] }
+            ],
+            conf_dir: '/conf/directory'
+          })
+      })
+    ) as jest.Mock;
+
+    render(<Configuration />);
+
+    await waitFor(() => screen.getByText('Empty configuration section'));
+    expect(screen.getByText('Empty configuration section')).toBeInTheDocument();
+  });
+
+  describe('ConfigurationKey Component', () => {
+    test('If the record has further values in it, should display record key, helpText as tooltip', () => {
+      const mockRecord = {
+        help: 'This is help text',
+        key: 'config.key',
+        is_anonymous: false,
+        values: [
+          {
+            help: 'Example config help text',
+            key: 'example.config',
+            is_anonymous: false,
+            value: 'Example value'
+          },
+          {
+            help: 'Another config help text',
+            key: 'another.config',
+            is_anonymous: false,
+            value: 'Another value'
+          }
+        ]
+      };
+
+      const { getByText, container } = render(<ConfigurationKey record={mockRecord} />);
+
+      expect(getByText(/config.key/i)).toBeInTheDocument();
+      const tooltip = container.querySelector('.config-tooltip');
+      expect(tooltip).toBeInTheDocument();
+    });
+  });
+
+  test('If the record has no further values in it, renders record key, its value if available, additional help text with no tooltip, default value marked as "Default:" if one exists', () => {
```

Comment:
Can you simplify this test name? Perhaps slit in multiple test if needed

Reply:
`  test('If the record has no further values in it, verifies the entire config key state', () => {.....}`